tests/fixtures/sauceswag-fail/sauce-runner.json

```diff
@@ -21,7 +21,10 @@
        {
           "name": "default",
           "browser": "chrome",
```

Comment:
have a discuss with Alex, it would be better to use `browserName` to tell apart from `browsers` and `browser`

Reply:
Sounds good! I'll change it to `browserName`.